test/e2e/serial/tests/scheduler_cache_stall.go

```diff
@@ -0,0 +1,371 @@
+
+			klog.Infof("using MCP %q - refresh period %v", mcpName, refreshPeriod)
+		})
+
+		When("there are jobs in the cluster", Label("job", "generic"), func() {
+			var idleJob *batchv1.Job
+			var hostsRequired int
+			var NUMAZonesRequired int
+			var expectedJobPodsPerNode int
+			var cpusPerPod int64 = 2 // must be even. Must be >= 2
+
+			BeforeEach(func() {
+				hostsRequired = 2
+				NUMAZonesRequired = 2
+
+				By(fmt.Sprintf("filtering available nodes with at least %d NUMA zones", NUMAZonesRequired))
+				nrtCandidates = e2enrt.FilterZoneCountEqual(nrtList.Items, NUMAZonesRequired)
+				if len(nrtCandidates) < hostsRequired {
+					e2efixture.Skipf(fxt, "not enough nodes with %d NUMA Zones: found %d", NUMAZonesRequired, len(nrtCandidates))
+				}
+				klog.Infof("Found %d nodes with %d NUMA zones", len(nrtCandidates), NUMAZonesRequired)
+
+				expectedJobPodsPerNode = 2 // anything >= 1 should be fine
+				idleJob = makeIdleJob(fxt.Namespace.Name, expectedJobPodsPerNode, len(nrtCandidates))
+
+				err := fxt.Client.Create(context.TODO(), idleJob) // will be removed by the fixture
+				Expect(err).ToNot(HaveOccurred())
+
+				_, err = wait.With(fxt.Client).Interval(3*time.Second).Timeout(30*time.Second).ForJobCompleted(context.TODO(), idleJob.Namespace, idleJob.Name)
+				Expect(err).ToNot(HaveOccurred())
+
+				// ensure foreign pods are reported
+				_, err = wait.With(fxt.Client).Interval(11*time.Second).Timeout(1*time.Minute).ForNodeResourceTopologiesSettled(context.TODO(), 3)
+				Expect(err).ToNot(HaveOccurred())
+			})
+
+			It("should be able to schedule pods with no stalls", func() {
+
+				desiredPodsPerNode := 2
+				desiredPods := hostsRequired * desiredPodsPerNode
+
+				// we use 2 Nodes each with 2 NUMA zones for practicality: this is the simplest scenario needed, which is also good
+				// for HW availability. Adding more nodes is trivial, consuming more NUMA zones is doable but requires careful re-evaluation.
+				// We want to run more pods that can be aligned correctly on nodes, considering pessimistic overreserve
+
+				Expect(desiredPods).To(BeNumerically(">", hostsRequired)) // this is more like a C assert. Should never ever fail.
+
+				// we can assume now all the zones from all the nodes are equal from cpu/memory resource perspective
+				referenceNode := nrtCandidates[0]
+				referenceZone := referenceNode.Zones[0]
+				cpuQty, ok := e2enrt.FindResourceAvailableByName(referenceZone.Resources, string(corev1.ResourceCPU))
+				Expect(ok).To(BeTrue(), "no CPU resource in zone %q node %q", referenceZone.Name, referenceNode.Name)
+
+				cpuNum, ok := cpuQty.AsInt64()
+				Expect(ok).To(BeTrue(), "invalid CPU resource in zone %q node %q: %v", referenceZone.Name, referenceNode.Name, cpuQty)
+
+				cpuPerPod := int(float64(cpuNum) * 0.6) // anything that consumes > 50% (because overreserve over 2 NUMA zones) is fine
+				memoryPerPod := 8 * 1024 * 1024 * 1024  // random non-zero amount
```

Comment:
just curious, why not simply 8Gi?

Reply:
no reason but the fact I didn't like to pass through `ParseQuantity` but in hindsight it would have been probably more readable